### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by coot**
* `ouroboros-network-api-0.4.0.0`
* `ouroboros-network-framework-0.6.0.0`
* `ouroboros-network-protocols-0.5.0.0`
* `ouroboros-network-0.7.0.0`
* `cardano-ping-0.2.0.0`

### CODE REVIEW

**Comment by coot** [comment]
This is needed in input-output-hk/ouroboros-consensus#56.

**Comment by coot** [comment]
bors merge

File: cardano-ping/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Revision history for cardano-ping
 
+## 0.2.0.0 -- 2023-05-08
+
+* Support for `NodeToNodeV_15` and `NodeToClientV_11`, e.g. support for
```
Comment (bolt12) [inline cardano-ping/CHANGELOG.md:5]: This should be NodeToNodeV_11 and NodeToClientV_15, you swapped the numbers :)

**Review by bolt12 — APPROVED** [review]
Just a small comment, I'll approve but needs change

File: cardano-ping/CHANGELOG.md (hunk 1 above)
Comment (coot) [reply cardano-ping/CHANGELOG.md]: Fixed.

File: ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs [hunk 2]
```diff
@@ -176,6 +181,46 @@ nodeToNodeCodecCBORTerm version
...
+                                  _ -> error "decodeTerm: impossible happened!",
+                  query = False
+                }
+          | x < 0 || x > 0xffffffff
+          = Left $ T.pack $ "networkMagic out of bound: " <> show x
+          | otherwise -- peerSharing < 0 || peerSharing > 2
+          = Left $ T.pack $ "peerSharing out of bound: " <> show peerSharing
+        decodeTerm _ t
+          = Left $ T.pack $ "unknown encoding: " ++ show t
+     in CodecCBORTerm {encodeTerm, decodeTerm = decodeTerm version }
```
Comment (bolt12) [inline ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs:223]: Is duplication necessary? can't one just do `query = query && version >= 12`?

**Review by bolt12 — APPROVED** [review]

File: ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs (hunk 2 above)
Comment (coot) [reply ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs:223]: That would work for `decode` but not for `encode`; this way will be also easier when we will remove unused versions after a HF.

Comment (coot) [reply ouroboros-network-api/src/Ouroboros/Network/NodeToNode/Version.hs:223]: ... for `encode` one would also need an `if` to not include the `TInt` in the list.

**Comment by coot** [comment]
bors merge

**Comment by coot** [comment]
bors merge

**Pull request merged**